packages/babel-helpers/src/helpers/applyDecs2311.ts

```diff
@@ -199,8 +199,16 @@ export default /* @no-mangle */ function applyDecs2311(
 ) {
   var symbolMetadata = Symbol.metadata || Symbol.for("Symbol.metadata");
   var defineProperty = Object.defineProperty;
+  var create = Object.create;
   var metadata: any;
-  var existingNonFields: Record<string, number> = {};
+  // Use both as and satisfies to ensure that we only use non-zero values
+  var existingNonFields = create(null) as Record<
```

Comment:
I thought about this, since we use `/` splicing it should be safe.
But I don't have a strong preference for it.

Reply:
I prefer this in case somebody injects stuff on the prototype.